content/guides/tina-cloud/existing-site/overview.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Overview
+last_edited: '2021-05-10T10:00:00.000Z'
+---
+
+In this guide, we will take you through adding Tina + Tina Cloud to a NextJS site.
```

Comment:
```suggestion
In this guide, we take you through adding TinaCMS and Tina Cloud to the Next.js blog starter.
With the help of TinaCMS, you will be able to edit your Next.js site in real time on an `/admin` route to avoid impacting your production website, and changes are saved back to your GitHub repository thanks to Tina Cloud Content API.
```